lms/djangoapps/mobile_api/testutils.py

```diff
@@ -123,8 +146,228 @@ def test_other_user(self):
         self.api_response(expected_response_code=404, username=other.username)
 
 
+class MobileAPIMilestonesMixin(object):
+    """
+    Tests the Mobile API decorators for milestones.
+
+    The two milestones supported in these tests are entrance exams and
+    pre-requisite courses. If either of these milestones are unfulfilled,
+    the mobile api will appropriately block content until the milestone is
+    fulfilled.
+    """
+    MILESTONE_ERROR = {'developer_message': 'Cannot access content with unfulfilled pre-requisites or unpassed entrance exam.'}  # pylint: disable=line-too-long
```

Comment:
Instead of using `disable=line-too-long`, why not break the string into 2 lines?